<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-01-17T22:22:25 IST" name="Default suite" finished-at="2025-01-17T22:22:41 IST" duration-ms="16068">
    <groups>
    </groups>
    <test started-at="2025-01-17T22:22:25 IST" name="Default test" finished-at="2025-01-17T22:22:41 IST" duration-ms="16068">
      <class name="dataDrivenTek.CreateOrganizationTest">
        <test-method is-config="true" signature="configBs()[pri:0, instance:dataDrivenTek.CreateOrganizationTest@49438269]" started-at="2025-01-17T22:22:25 IST" name="configBs" finished-at="2025-01-17T22:22:25 IST" duration-ms="21" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configBs -->
        <test-method is-config="true" signature="configBC()[pri:0, instance:dataDrivenTek.CreateOrganizationTest@49438269]" started-at="2025-01-17T22:22:25 IST" name="configBC" finished-at="2025-01-17T22:22:29 IST" duration-ms="3991" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configBC -->
        <test-method is-config="true" signature="configBM()[pri:0, instance:dataDrivenTek.CreateOrganizationTest@49438269]" started-at="2025-01-17T22:22:29 IST" name="configBM" finished-at="2025-01-17T22:22:35 IST" duration-ms="5670" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configBM -->
        <test-method signature="createOrganizationtest()[pri:0, instance:dataDrivenTek.CreateOrganizationTest@49438269]" started-at="2025-01-17T22:22:35 IST" name="createOrganizationtest" finished-at="2025-01-17T22:22:35 IST" duration-ms="96" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because the return value of "com.comcast.crm.genericwebdriverutility.UtilityClassObject.getTest()" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentTest.log(com.aventstack.extentreports.Status, String)" because the return value of "com.comcast.crm.genericwebdriverutility.UtilityClassObject.getTest()" is null
at dataDrivenTek.CreateOrganizationTest.createOrganizationtest(CreateOrganizationTest.java:42)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createOrganizationtest -->
        <test-method is-config="true" signature="configAM()[pri:0, instance:dataDrivenTek.CreateOrganizationTest@49438269]" started-at="2025-01-17T22:22:35 IST" name="configAM" finished-at="2025-01-17T22:22:41 IST" duration-ms="5954" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configAM -->
        <test-method is-config="true" signature="AS()[pri:0, instance:dataDrivenTek.CreateOrganizationTest@49438269]" started-at="2025-01-17T22:22:42 IST" name="AS" finished-at="2025-01-17T22:22:42 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AS -->
      </class> <!-- dataDrivenTek.CreateOrganizationTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
